src/datepicker/datepicker.js

```diff
@@ -212,6 +214,11 @@ angular.module('ui.bootstrap.datepicker', ['ui.bootstrap.dateparser', 'ui.bootst
       date = dateParser.fromTimezone(date, ngModelOptions.timezone);
       ngModelCtrl.$setValidity('dateDisabled', !date ||
         this.element && !this.isDisabled(date));
+
+      if ($scope.maintainFocus) {
+        $scope.$broadcast('uib:datepicker.focus');
```

Comment:
who listens to this?

Reply:
The datepicker does.  They use it mostly so that the popover can focus on open.  I could call the focus method directly, but figured I'd re-use the event

Reply:
Thats probably fine, the only downside is this leaks an extra event out to all consumers